pkg/ibmvpc/analysis_output_test.go

```diff
@@ -671,6 +671,20 @@ var tests = []*vpcGeneralTest{
 		useCases:    []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
 		format:      vpcmodel.Text,
 	},
+	{
+		inputConfig:  "filters_split_lb_subnet",
+		useCases:     []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
+		format:       vpcmodel.Text,
+		grouping:     false,
+		noLbAbstract: true,
+	},
+	{
+		inputConfig:  "filters_split_lb_subnet",
+		useCases:     []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
+		format:       vpcmodel.HTML,
+		grouping:     true,
+		noLbAbstract: true,
+	},
```

Comment:
Do we have a test to a block with no free addresses?

Reply:
yes